backend/app/models/user.rb

```diff
@@ -40,11 +42,11 @@ def drafts
   end
 
   def admin
-    role == 'admin'
+    admin?
   end
 
   def staff
-    role == 'staff'
+    author?
```

Comment:
`author` is not equal to `staff` role. staff role was meant to mean `editor` or `moderator` so using `editor?` reflect it better.